tree/ntuple/v7/test/ntuple_evolution.cxx

```diff
@@ -579,7 +579,7 @@ TEST(RNTupleEvolution, AddedIntermediateClass)
 {
    FileRaii fileGuard("test_ntuple_evolution_added_intermediate_class.root");
 
-   WriteOldInFork([&] {
+   ExecInFork([&] {
```

Comment:
has to go into the commit that renamed the functions

Reply:
still needs to go into the commit that renamed the function

Reply:
I was sure I moved this, no idea how did this slip through.